src/services/renderer.service.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable, PLATFORM_ID, Inject, ElementRef } from '@angular/core';
 import { isPlatformBrowser } from '@angular/common';
 
 @Injectable()
-export class Renderer {
+export class Renderer2 {
```

Comment:
This feels like Angular's upgrade script going rogue. It was supposed to replace all instances of the Renderer class with the new Renderer2 class... I'm not so sure it was supposed to re-name _our_ Renderer class to be Renderer2

Also, what's this class doing here? I'm not 100% sure, looks like it was added when upgrading to Angular 6, but... why?

Reply:
Isn't our "Renderer" class overloading or extending the built in Renderer class?  This was a manual change on my part attempting to follow the instruction:
"Anywhere you use Renderer, now use Renderer2"
As to why this is here, I'm not entirely sure.  For this project, I was going to focus on getting everything up to date first and then do a comprehensive review of what is where, why, and if it can be removed.

Reply:
This class isn't extending angular's Renderer class. It's a completely separate class. (Extension in typescript would require the `extends` keyword.)

Reply:
Ah gotcha.  In that case, I can reverse these changes.

Reply:
Reverted in https://github.com/RenovoSolutions/ngx-datetimepicker/pull/146/commits/4f43868eaff032d38100875e8dfae04648a7fbed